WD-Firmware/rtos/rtosal/src/rtosal_memory.c

```diff
@@ -67,5 +67,7 @@ u32_t rtosalMsgQueueRecieve (rtosalMsgQueue_t* pRtosalMsgQueueCb, void* pRtosalM
 
 #elif D_USE_THREADX
 
+#else
```

Comment:
remove this function, it is by mistake here....
Nati, write down for yourself a note to implement memory apis